rudder-web/src/main/webapp/javascript/rudder/rudder.js

```diff
@@ -69,7 +69,7 @@ var bootstrapScrollspy = $.fn.scrollspy.noConflict();
 var bootstrapTab = $.fn.tab.noConflict();
 var bootstrapAffix = $.fn.affix.noConflict();
 var bootstrapModal = $.fn.modal.noConflict();
-$.fn.bsModal = bootstrapModal
+$.fn.bsModal = bootstrapModal;
```

Comment:
Maybe we can remove that et now use modal function directly, but you would have to rewrite too much things so keep it :)